.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        device: ["xc7a35t", "xc7a100t", "xc7a200t", "xc7z010", "xczu7ev", "xc7k480t", "xc7vx980t", "LIFCL-40", "testarch"]
+        device: ["xc7a35t", "xc7a100t", "xc7a200t", "xc7s50", "xc7z010", "xczu7ev", "xc7k480t", "xc7vx980t", "LIFCL-40", "testarch"]
```

Comment:
`xc7s25` seems to be missing here, unless there was a reason to leave it out

Reply:
As I mentioned in PR description, `xc7s25` fails due to a missing part in Vivado. I think it's caused by incompatible Vivado version.

Reply:
Ah sorry, I somehow skipped those lines. 

What version are you using locally? as GH actions CI uses Vivado 2017.2 and we are in the process of adding 2019.2.

Reply:
Ok, so once we move CI to use 2019.2 we might test `xc7s25` as well

Reply:
The Vivado version used in CI complained that it does not know such a part. That's why it is disabled for now.